debian/marquee/nginx/sites/books.spec.whatwg.org.conf

```diff
@@ -0,0 +1,13 @@
+server {
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+
+    server_name books.spec.whatwg.org;
+
+    ssl_certificate /etc/letsencrypt/live/books.spec.whatwg.org/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/books.spec.whatwg.org/privkey.pem;
+
+    include /etc/nginx/whatwg.conf;
+
+    return 301 https://books.idea.whatwg.org$request_uri;
```

Comment:
For idea↔spec redirects I think we should use HTTP 302, as 301 may be cached, and we might move things between. This was a bit of a nuisance with https://whatwg.org/faq.